scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -530,6 +532,8 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             File.Delete(compilersDeps);
             File.Delete(compilersRuntimeConfig);
 
+            CrossgenUtil.CrossgenDirectory(c, outputDir);
```

Comment:
@piotrpMSFT originally set up the crossgenning of stage1 and we saw dramatic performance improvement.